src/robot/libdocpkg/consoleviewer.py

```diff
@@ -74,7 +72,7 @@ def _show_inits(self, lib):
     def _show_keyword(self, kw, show_name=True):
         if show_name:
             self._header(kw.name, underline='-')
-        self._data([('Arguments', '[%s]' % ', '.join(kw.args))])
+        self._data([('Arguments', '[%s]' % unicode(kw.args))])
```

Comment:
`unicode()` could be omitted by using `u'[%s]'` as the template. This is fine as well.